coconut/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+#[macro_use]
+mod error;
+pub mod issue;
+pub mod lagrange;
+pub mod request;
+pub mod setup;
+
+#[cfg(test)]
+#[path = "tests/fixtures.rs"]
+mod fixtures;
+
+#[cfg(test)]
+#[path = "tests/integration_tests.rs"]
+mod integration_tests;
```

Comment:
Classically, "integration tests" are supposed to use the public APIs of the package and be located in `[crate]/tests`:
https://doc.rust-lang.org/rust-by-example/testing/integration_testing.html#:~:text=Unit%20tests%20are%20testing%20one,your%20library%20work%20correctly%20together.

Reply:
Rename it to end_to_end?

Reply:
The point was: do you need private APIs in your integration tests? Normally no, so the default location of cargo for (integration) tests apply.